src/theme/components/button.ts

```diff
@@ -0,0 +1,37 @@
+import { DefaultTheme } from '..';
+
+const getButtonTheme = (theme: DefaultTheme) => ({
+  background: {
+    danger: theme.palette.danger.shade5,
+    dangerHover: theme.palette.danger.shade6,
```

Comment:
Should hover be a nested property?

`background: {
   primary: ...
   secondary: ...
  ...
   hover: {
      primary: ...
      ...
   }
}`